com.chenyifaer.FaForever.appdata.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Generator for metainfo & .desktop files:
+    https://www.freedesktop.org/software/appstream/metainfocreator/#/
+-->
+<component type="desktop-application">
+    <id>com.chenyifaer.FaForever</id>
+    <name>FaForever</name>
+    <summary>A third-party music player that replaces Synology DS Audio client.</summary>
+    <developer_name>kjxbyz</developer_name>
+    <url type="homepage">https://chenyifaer.com/faforever</url>
+    <metadata_license>MIT</metadata_license>
+    <project_license>MIT</project_license>
+    <supports>
+        <control>pointing</control>
+        <control>keyboard</control>
+    </supports>
+    <recommends>
+        <control>touch</control>
+    </recommends>
+    <description>
+        <p>A third-party music player that replaces Synology DS Audio client.</p>
+        <p>Features</p>
+        <ul>
+            <li>Open Source!</li>
+        </ul>
+    </description>
+    <launchable type="desktop-id">com.chenyifaer.FaForever.desktop</launchable>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/cyf/faforever/main/screenshots/desktop.png</image>
```

Comment:
you should never reference a branch.

(same with the other screenshots)

Reply:
Can tags be used instead?

Reply:
Yea

Reply:
> Can tags be used instead?

Yes